pkgs/applications/networking/mullvad/default.nix

```diff
@@ -5,5 +5,4 @@
 lib.makeScope newScope (self: {
   libwg = self.callPackage ./libwg.nix { };
   mullvad = self.callPackage ./mullvad.nix { };
-  openvpn-mullvad = self.callPackage ./openvpn.nix { };
```

Comment:
IMO the OpenVPN removal is a bit premature -- I'd be comfortable removing it after the next NixOS release (25.11), but would rather not break users relying on it sooner than necessary.